pkg/bucketclass/bucketclass_suite_test.go

```diff
@@ -0,0 +1,13 @@
+package bucketclass_test
```

Comment:
Actually these tests were kept in the same package as the testing functions so that the non-exported functions could also be tested. A lot of tests are actually testing the internal functions only. In order to move the tests to another package, we can start exporting the internal functions as well.

Reply:
👍